src/structured_config.py

```diff
@@ -110,6 +112,20 @@ def log_cleaner_delete_retention_ms_validator(cls, value: str) -> Optional[int]:
             return int_value
         raise ValueError("Value out of range.")
 
+    @validator("ssl_principal_mapping_rules")
+    @classmethod
+    def ssl_principal_mapping_rules_validator(cls, value: str) -> Optional[str]:
+        """Check that the list is formed by valid regex values."""
+        # get all regex up until replacement position "/"
+        pat = re.compile(r"RULE:([^/]+)(?:,RULE:[^/]+)*(?:DEFAULT){0,1}")
```

Comment:
**praise:** Really elegant and clean implementation. 

I noticed it would break with `RULE:123/,RULE:ABC,DEFAULT`, but maybe just add a 'TODO' here for 'missing replacement position checks' to spare the sanity.

Otherwise, really clean, love it!

Reply:
wouldn't that be valid as an empty replacement? I'm unsure though as the internals of this

Reply:
Uhm, that would look a bit weird to me too. I believe that does not quite comply with the one raul provided in the description of the field